Source/WTF/wtf/text/StringCommon.h

```diff
@@ -65,6 +65,23 @@ inline std::span<const char> span(const char* string)
     return unsafeMakeSpan(string, string ? strlen(string) : 0);
 }
 
+inline std::span<const LChar> span(const LChar* string)
+{
+    return unsafeMakeSpan(string, string ? strlen(byteCast<char>(string)) : 0);
+}
+
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+inline std::span<const UChar> span(const UChar* string)
```

Comment:
I guess in the long run to make this safe we’ll need a type to distinguish “null terminated pointer” from other kinds of pointer.

Reply:
`ASCIILiteral.h` has a similar issue as I recall.